src/routing/Page.tsx

```diff
@@ -12,11 +12,20 @@ import { RouterContext } from './RouterContext';
 export default function Page(props: PageProps): JSX.Element {
   const { values, submitForm } = useFormikContext();
   const formValues = values as IFormData;
-  const listOfRoutes = useContext(RouterContext).listOfRoutes;
+  const { listOfRoutes, updateRoute } = useContext(RouterContext);
   const currentLocation = useLocation();
 
+  // const history = useHistory();
+  // updateRoute(currentLocation.pathname);
+
   const navigate = useNavigate();
 
+  useLayoutEffect(() =>
```

Comment:
is there a reason to use `useLayoutEffect` instead of `useEffect` here?

Reply:
It's really strange, but I see this error but only when I'm in debug mode: 
<img width="906" alt="Screen Shot 2022-05-26 at 3 32 35 PM" src="https://user-images.githubusercontent.com/2727868/170590998-79253a8b-11a5-43bd-9273-796e28050ae8.png"> 
The error does not show up when using useLayoutEffect. I think I had switched one of them back and was experimenting with the rendering of both, so the fact that chapter and page have different uses was a mistake. I may switch back to useEffect and just warn the team that if they're in debug mode, they may see this error, but it doesn't seem to really effect anything. I think it's a bug in react.